README.md

```diff
@@ -35,6 +35,7 @@ Introduce commercial and open source tools for memory acquisition.
 - [FEX Memory Imager](https://getdataforensics.com/product/fex-memory-imager/) - FEX Memory Imager (FEX Memory) is a free imaging tool designed to capture the physical Random Access Memory (RAM) of a suspect's running computer. This allows investigators to recover and analyze valuable artifacts found only in memory.
 - [MacQuisition](https://www.blackbagtech.com/category/blog/macquisition/)
 - [Digital Collector](https://cellebrite.com/en/digital-collector/) - A powerful forensic imaging software solution to perform triage, live data acquisition and targeted data collection for Windows and Mac computers.
+- [varc (Volatile Artifact Collector)](https://github.com/cado-security/varc) - varc collects a snapshot of volatile data from a system.
```

Comment:
Does we need to use a full name? For now, I would like to put the tool name in front of the sentence for clarity.

```suggestion
- [varc](https://github.com/cado-security/varc) - varc collects a snapshot of volatile data from a system.
```